lib/commands/upgrade.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
+const path          = require('path');
+const Logger        = require('../logger');
```

Comment:
You can require relative to root directory
`require('lib/logger');'

Same for the rest of the files